packages/ng/src/app/select/searcher/select-searcher-intl.ts

```diff
@@ -0,0 +1,32 @@
+import {Injectable, SkipSelf, Optional } from '@angular/core';
```

Comment:
if you wanna use some files to store this kind of code, you can use `.intl` and not `-intl`

Reply:
it respects way more the naming convention of `xxx.yyy(.spec).ext`, and yu'll have things like that
```
select.component.ts
select.component.spec.ts
select.module.ts
select.intl.ts
```

i know what is what at a glance

Reply:
for intl, you think that `.intl` is better than `-intl` ? why not